libethereum/ExtVM.cpp

```diff
@@ -95,30 +95,94 @@ void go(unsigned _depth, Executive& _e, OnOpFunc const& _onOp)
 
 bool ExtVM::call(CallParameters& _p)
 {
-	Executive e(m_s, envInfo(), m_sealEngine, depth + 1);
+	m_successfulCalls.emplace_back(m_s, envInfo(), m_sealEngine, depth + 1);
```

Comment:
Can you add a comment why we first add it to successful calls and remove it again in case it was not successful instead of just adding it when it returned successfully?